spi/src/main/java/org/arquillian/cube/spi/metadata/HasMappedPorts.java

```diff
@@ -0,0 +1,13 @@
+package org.arquillian.cube.spi.metadata;
+
+
+public interface HasMappedPorts extends CubeMetadata {
+    
+    String getIP();
+
+    Integer forTargetPort(int targetPort);
+
+    void createProxies() throws Exception;
```

Comment:
I don't see that.  We need to create the proxies after the pod has started, so we need a listener somewhere that does that before running the tests.  What am I missing?